boltdesignsystem/bolt
# DS-44: Convert Schemas

## Jira

https://pegadigitalit.atlassian.net/browse/DS-44

## Summary

Converted all the .yml schemas to .js schemas

## Details

Replaced the old .yml schemas with the modern .js schemas and updated all the dependencies in the Pattern Lab build and within the components. 

## How to test

1. Navigate to https://feature-bds-44.boltdesignsystem.com/pattern-lab/?p=components-overview
2. Check to confirm that the schemas values are being rendered within the schema tables
3. Pull down the latest and confirm that 'yarn start' is working correctly.
